scripts/add-sentry-to-vlc.patch

```diff
@@ -1,40 +1,162 @@
-index 45e05469..0050ffbc 100644
---- a/Sources/VLCAppDelegate.m
-+++ b/Sources/VLCAppDelegate.m
-@@ -23,6 +23,7 @@
- #import <AppCenter/AppCenter.h>
- #import <AppCenterAnalytics/AppCenterAnalytics.h>
- #import <AppCenterCrashes/AppCenterCrashes.h>
+   pod 'SwiftLint', '~> 0.47.1', :configurations => ['Debug']
+diff --git a/Sources/App/iOS/VLCAppDelegate.m b/Sources/App/iOS/VLCAppDelegate.m
+index 2c1fc802..21495fd7 100644
+--- a/Sources/App/iOS/VLCAppDelegate.m
++++ b/Sources/App/iOS/VLCAppDelegate.m
+@@ -19,6 +19,7 @@
+ 
+ #import "VLCAppDelegate.h"
+ #import "VLC-Swift.h"
 +#import <Sentry/Sentry.h>
  
- NSString *VLCAppCenterAppID = @"0114ca8e-2652-44ce-588d-2ebd035c3577";
+ @interface VLCAppDelegate ()
+ {
+@@ -134,6 +135,16 @@
  
-@@ -136,6 +137,14 @@ NSString *VLCAppCenterAppID = @"0114ca8e-2652-44ce-588d-2ebd035c3577";
  - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
  {
-     [MSACAppCenter start:VLCAppCenterAppID withServices:@[[MSACAnalytics class], [MSACCrashes class]]];
-+    
 +    [SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
 +        options.dsn = @"https://a92d50327ac74b8b9aa4ea80eccfb267@o447951.ingest.sentry.io/5428557";
 +        options.environment = @"integration-tests";
-+        options.tracesSampleRate = @0;
++        options.tracesSampleRate = @1.0;
 +        options.enableFileIOTracking = YES;
-+        options.profilesSampleRate = @1.0;
++        options.attachScreenshot = YES;
++        options.attachViewHierarchy = YES;
++        options.enableUserInteractionTracing = YES;
 +    }];
- 
++    
      self.orientationLock = UIInterfaceOrientationMaskPortrait | UIInterfaceOrientationMaskLandscape;
  
+     self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+diff --git a/VLC.xcodeproj/project.pbxproj b/VLC.xcodeproj/project.pbxproj
+index 311b5c8b..b434a8b5 100644
+--- a/VLC.xcodeproj/project.pbxproj
++++ b/VLC.xcodeproj/project.pbxproj
+@@ -492,7 +492,7 @@
+ 		4152F1611FEF19BD00F1908B /* KeychainCoordinator.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = KeychainCoordinator.swift; sourceTree = "<group>"; };
+ 		41533C91211338D500EC3ABA /* VLC-iOSTests.xctest */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; path = "VLC-iOSTests.xctest"; sourceTree = BUILT_PRODUCTS_DIR; };
+ 		41533C9D2113392F00EC3ABA /* URLHandlerTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = URLHandlerTests.swift; sourceTree = "<group>"; };
+-		41533CA1211343D100EC3ABA /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = Info.plist; path = ../../Testing/Unit/Info.plist; sourceTree = "<group>"; };
++		41533CA1211343D100EC3ABA /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = Info.plist; path = ../../Buildsystem/Testing/Unit/Info.plist; sourceTree = "<group>"; };
```

Comment:
I had to fix some paths to make the UITests compile. I opened a PR for this on the VLC repo https://github.com/videolan/vlc-ios/pull/452